sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -249,80 +278,76 @@ for document in result:
 ```
 
 ### Translate multiple inputs
+
 Begin translating with documents in multiple source containers to multiple target containers in different languages.
 
+<!-- SNIPPET:sample_begin_translation.begin_translation -->
```

Comment:
This one should show translating multiple inputs. Should it use `SNIPPET:sample_translate_multiple_inputs.multiple_translation`?